.github/workflows/.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}

```

Comment:
- [ ] [OPTIONAL] Please make sure to remove this file, this folder `.github/workflows` is supposed to only host your `linters.yml` file which is the _only_ config file that you need in order to set up linters on GitHub, `.hintrc` is a local config file that is used to set up linters on your local machine and should be in the root directory if you wish to add to your repository.

Reply:
@ouasamine Thanks! you are right. the folder "workflows" only require the file linters.yml but the previus reviewer asked to includes those files, the one called ".hintrc" and the other one.
but yes I knew that they were not necessary, I only did it because I was told to do so.
but thanks for your comment :)